fabric-network-v1/src/main/java/net/fabricmc/fabric/api/network/v1/receiver/PacketReceiverRegistry.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.network.v1.receiver;
+
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.fabric.api.network.v1.receiver.PacketContext.ClientLoginQuery;
+import net.fabricmc.fabric.api.network.v1.receiver.PacketContext.ClientPlay;
+import net.fabricmc.fabric.api.network.v1.receiver.PacketContext.ServerPlay;
+import net.fabricmc.fabric.impl.network.receiver.SimplePacketReceivers;
+
+public interface PacketReceiverRegistry<T extends PacketContext> {
+	PacketReceiverRegistry<ServerPlay> SERVER_PLAY = SimplePacketReceivers.SERVER_PLAY;
```

Comment:
Maybe add some comments to these fields to specify if the packets from these are received on the Server/Client.